contexts/arbo-context.tsx

```diff
@@ -47,51 +47,35 @@ function filterData(data: any[], filters: { [key: string]: string[] }): any[] {
     return filterKeys.every((key: string) => {
       if (!filters[key].length) return true;
 
-      if(key === "end_date") {
-        const filterEndDate = parseISO(filters["end_date"][0]);
+      if (key === "end_date") {
+        const filterEndDate = new Date(filters["end_date"][0]);
```

Comment:
I'd like to keep the `parseISO` here so that we don't run into locale related bugs again. If there's a particular reason we need to use `new Date` here, let's fix that underlying issue and store iso strings instead.

Reply:
Fro sure will attempt to use parseISO i just kept running into issues with it but maybe that was because I was not consistent everywhere with passing the dates.